README.md

```diff
@@ -22,7 +22,7 @@ Simply add an instance of `CSSSplitWebpackPlugin` to your list of plugins in you
 
 ```javascript
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
-var CSSSplitWebpackPlugin = require('../').default;
+var CSSSplitWebpackPlugin = require('css-split-webpack-plugin');
```

Comment:
I think this needs to be `require('css-split-webpack-plugin').default`

Reply:
Fixed...